tests/conftest.py

```diff
@@ -334,7 +334,7 @@ def inner(event):
             by_parent.setdefault(span["parent_span_id"], []).append(span)
 
         def render_span(span):
-            yield "- op={!r}: description={!r}".format(
+            yield "- op='{}': description='{}'".format(
```

Comment:
What's the motivation to change this?

Reply:
It caused incompatibilities between Python 2 and 3 when rendering unicode strings.